Sources/AnyLint/Lint.swift

```diff
@@ -141,7 +141,35 @@
             return
         }
 
-        Statistics.shared.found(violations: customClosure(checkInfo), in: checkInfo)
+        Statistics.shared.found(violations: [checkInfo: customClosure(checkInfo)])
+    }
+    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Lines should not have trailing whitespace.](https://app.codacy.com/gh/Flinesoft/AnyLint/pullRequest?prid=5691372)